src/cards/light-card/light-card-config.ts

```diff
@@ -25,6 +25,7 @@ export const lightCardConfigStruct = assign(
         hide_state: optional(boolean()),
         show_brightness_control: optional(boolean()),
         show_color_temp_control: optional(boolean()),
+        show_color_control: optional(boolean()),
```

Comment:
```suggestion
        show_color_control: optional(boolean()),
        use_light_icon_color: optional(boolean()),
```
I think we can split the 2 features for people who don't want colored icon by color control ?